src/libstd/num/f32.rs

```diff
@@ -126,15 +126,6 @@ mod cmath {
 #[cfg(not(test))]
 #[lang = "f32"]
 impl f32 {
-    /// Parses a float as with a given radix
-    #[unstable(feature = "float_from_str_radix", reason = "recently moved API",
-               issue = "27736")]
-    #[rustc_deprecated(since = "1.4.0",
-                 reason = "unclear how useful or correct this is")]
-    pub fn from_str_radix(s: &str, radix: u32) -> Result<f32, ParseFloatError> {
-        num::Float::from_str_radix(s, radix)
```

Comment:
I see that the actual implementation in `core::num` is still around. I assume this is to avoid dealing with the fallout, e.g. in the tests using `from_str_radix("...", 16)` later in this file? (By the way, don't those need to be updated to call `Float::from_str_radix`?)